Source Code/Ticket Management System/customer_menu.c

```diff
@@ -105,8 +110,38 @@ void viewFAQ()
 }
 
 void updateContactInfo()
-{
-	// TODO
+{	
+	system("CLS");
```

Comment:
IMHO it would be more convenient using the `menu` and `menuOption` system.

Reply:
I assumed as much when I was writing it, but the code you used in the Auth menu is quite complicated for how one dimentional this operation is, you either update the fields or you don't I am not so sure about the "no" input, I will keep it as it is now but it will be open to refactoring it as a menu latter if somebody wants to..

Reply:
fixed